.circleci/config.yml

```diff
@@ -54,6 +56,28 @@ jobs:
           command: bash <(curl -s https://codecov.io/bash)
           when: always
 
+  kotlin-smoke-test:
+    machine: true
+    steps:
+      - checkout
+      - run:
+          name: Get kotlin generator
+          command: docker pull gcr.io/kotlin-gapic/kgen
```

Comment:
@jbolinger I am having a hard time finding the appropriate tag to use for this docker image. Which should I use?

Reply:
Are you able to access this page? https://console.cloud.google.com/gcr/images/kotlin-gapic/GLOBAL/kgen

You can see the tags there (I suspect that we'll move things to docker hub as things mature). You can pin to a recent tag or try to stay up to date with latest. I try not to break things but it's early days :)